src/coreclr/inc/sospriv.idl

```diff
@@ -444,7 +444,7 @@ interface ISOSDacInterface8 : IUnknown
 // Increment anytime there is a change in the data structures that SOS depends on like
 // stress log structs (StressMsg, StressLogChunck, ThreadStressLog, etc), exception
 // stack traces (StackTraceElement), the PredefinedTlsSlots enums, etc.
-cpp_quote("#define SOS_BREAKING_CHANGE_VERSION 4")
+cpp_quote("#define SOS_BREAKING_CHANGE_VERSION 5")
```

Comment:
cc @mikem8361 `SOS_BREAKING_CHANGE_VERSION` bump

Reply:
This is good, but we need to bump the version to 5 in the diagnostics repo.  We should have done that when Jan updated SOS for the Exception's _stackTrace changes.

Reply:
Ah, I didn't know there is a counterpart for this in the diag repo.

Reply:
I should have caught this in the diag PR. I have a PR up to make this change and hopefully get into our pending tools release.